tests/test_onnx.py

```diff
@@ -38,19 +38,23 @@ def forward(self, input_ids, some_other_args, token_type_ids, attention_mask):
 
 
 class OnnxExportTestCase(unittest.TestCase):
-    MODEL_TO_TEST = ["bert-base-cased", "gpt2", "roberta-base"]
```

Comment:
Removing `roberta-base` is on purpose here?

Reply:
Yeah, speeding up things a bit, Roberta and Bert share the exact same graph, so basically it's testing the same things twice.